corehq/toggles.py

```diff
@@ -5,13 +5,43 @@
 import math
 from toggle.shortcuts import toggle_enabled, set_toggle
 
-Tag = namedtuple('Tag', 'name css_class')
-TAG_ONE_OFF = Tag(name='One-Off', css_class='danger')
-TAG_EXPERIMENTAL = Tag(name='Experimental', css_class='warning')
-TAG_PRODUCT_PATH = Tag(name='Product Path', css_class='info')
-TAG_PRODUCT_CORE = Tag(name='Core Product', css_class='success')
-TAG_PREVIEW = Tag(name='Preview', css_class='default')
-TAG_UNKNOWN = Tag(name='Unknown', css_class='default')
+Tag = namedtuple('Tag', 'name css_class description')
+TAG_ONE_OFF = Tag(
+    name='One-Off',
+    css_class='danger',
+    description="This feature flag was created for one specific use-case. "
+    "Please don't enable it for your project without first talking to the tech "
+    "team. This is not fully supported and may break other features.",
+)
+TAG_EXPERIMENTAL = Tag(
+    name='Experimental',
+    css_class='warning',
+    description="This feature flag is a proof-of-concept that we're currently "
+    "testing out. It may be changed before it is released or it may be dropped.",
+)
+TAG_PRODUCT_PATH = Tag(
+    name='Product Path',
+    css_class='info',
+    description="We intend to release this feature.  It may still be in QA or "
+    "we may have a few changes to make before it's ready for general use.",
+)
+TAG_PRODUCT_CORE = Tag(
+    name='Core Product',
+    css_class='success',
+    description="This is a core-product feature that you should feel free to "
+    "use.  We've feature-flagged it probably because it is an advanced "
+    "workflow we'd like more control over.",
+)
+TAG_PREVIEW = Tag(
+    name='Preview',
+    css_class='default',
+    description="",  # I'm not sure...
+)
```

Comment:
What does it mean for a feature to be flagged as "Preview", exactly?  Looks like currently this is only enabled for `Access to map-based demo reports`.  I propose we drop it.  Can we also drop the "Unknown" tag?